### PR TITLE
BD-836 Compliance Issues in Metrological RDKSplashScreen Source

### PR DESCRIPTION
**Pull request by Sukanya673**
BD-836 Compliance Issues in Metrological RDKSplashScreen Source

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2019 Metrological
```
Comment (mhughesacn) [inline LICENSE:189]: Please make this 2020 so all the dates are consistent.

**Comment by mhughesacn** [comment]
Approved, thank you

**Comment by pradeeptakdas** [comment]
Hi @wouterlucas  , Please merge this change if you are happy, Thanks!

**Comment by wouterlucas** [comment]
LGTM!

**Pull request merged**